js/script.js

```diff
@@ -297,6 +297,35 @@ document.addEventListener('click', (e) => {
   }
 });
 
+/*
+ Email Validation
+ */
+ document.querySelector('#contact-form').addEventListener('submit', (e) => {
+  let valid = true;
 
+  const responseContainer = document.querySelector('#response-text');
+
+  const email = document.querySelector('#email').value;
+
+  const emailArray = email.split('');
+
+  for (let i = 0; i < emailArray.length; i += 1) {
+    if (
+      emailArray[i] !== emailArray[i].toLowerCase()
+      && emailArray[i] !== '@' && emailArray[i] !== '.'
+    ) {
+      valid = false;
+    }
+  }
+
+  // if the email is valid, proceed with form submission
+  if (valid) {
+    // empty the response test in contact form
+    responseContainer.innerHTML = '';
+  } else {
+    e.preventDefault();
+    responseContainer.innerHTML = 'Invalid Email';
+  }
+});
```

Comment:
- [ ] The email verification is not working as expected. I inputted an uppercase email then the form got submitted without any errors. Please work on this validation to check if the email is in lower case.

See test screenshot

Filling the form | After Submitting |
:------------------:|:----------------------:|
![a1](https://user-images.githubusercontent.com/39523312/157518507-ed2b7197-6539-480e-9116-013e580263a6.png)| ![a2](https://user-images.githubusercontent.com/39523312/157518521-14fc4ed5-be32-4071-8839-836a6e60ca75.png)